modules/contrib_rules_jvm/0.21.4/MODULE.bazel

```diff
@@ -0,0 +1,249 @@
+module(
+    name = "contrib_rules_jvm",
+    version = "0.21.4",
+    repo_name = "contrib_rules_jvm",
+)
+
+# TODO: This should be read from repositories.bzl, but we can't until this issue is solved:
+# - https://github.com/bazelbuild/bazel/issues/17880
+PROTOBUF_VERSION = "21.7"
+
+# The java packages are published to maven under a different versioning scheme.
+PROTOBUF_JAVA_VERSION = "3.{}".format(PROTOBUF_VERSION)
+
+bazel_dep(
+    name = "apple_rules_lint",
+    version = "0.3.2",
+)
+bazel_dep(
+    name = "bazel_skylib",
+    version = "1.4.2",
+)
+bazel_dep(
+    name = "bazel_skylib_gazelle_plugin",
```

Comment:
Now that everything is passing, time for nits (I will merge anyway) :-)

Does this really have to be a non-dev dependency? The same question applies for some of the other deps.

Reply:
https://github.com/bazel-contrib/rules_jvm/pull/237 :)